containers/dotnetcore-mssql/.devcontainer/mssql/setup-db.sh

```diff
@@ -0,0 +1,36 @@
+sleep 60s
```

Comment:
Could this setup-db.sh script be invoked as an option "postCreateCommand" script from devcontainer.json instead of an entrypoint? That way it doesn't need to be baked into the image which should speed up start time.

Reply:
I've updated the setup process to a postCreateCommand script, thanks!